### PR TITLE
docs: Update main branch Warning with standard text agreed to by TSC

### PR DESCRIPTION
**Pull request by lenny-goodell**
**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 > **Warning**  
-> The `main` branch contains work-in-progress development code for the upcoming EdgeX 3.0.0 release, **and is not guaranteed to be stable or working**. It is only compatible with the [main branch of edgex-compose](https://github.com/edgexfoundry/edgex-compose).
+> The **main** branch of this repository contains work-in-progress development code for the upcoming release, and is **not guaranteed to be stable or working**.
+> It is only compatible with the [main branch of edgex-compose](https://github.com/edgexfoundry/edgex-compose) which uses the Docker images built from the **main** branch of this repo and other repos.
 >
-> **The current stable branch of device-onvif-camera is [Levski](https://github.com/edgexfoundry/device-onvif-camera/tree/levski).**
+> **The source for the latest release can be found at [Releases](https://github.com/edgexfoundry/device-onvif-camera/releases).**
 
 ### General
 Visit the [EdgeX documentation page](https://docs.edgexfoundry.org/3.0/microservices/device/supported/device-onvif-camera/General/) to learn more about the latest release.
```
Comment (ChristianDarr-personal) [inline README.md]:
```suggestion
Visit the [EdgeX documentation page](https://docs.edgexfoundry.org/latest/microservices/device/supported/device-onvif-camera/General/) to learn more about the latest release.
```
This is related to the comments left on the usb [remove-docs pr](https://github.com/edgexfoundry/device-usb-camera/pull/238), but I think this is relevant to this pr.

**Review by ChristianDarr-personal — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lenny-goodell) [reply README.md]: updated.

**Review by ChristianDarr-personal — APPROVED** [review]
LGTM

**Review by presatish — APPROVED** [review]
LGTM

**Pull request merged**